signing_service/signing_service/signing_service.py

```diff
@@ -136,6 +180,11 @@ def _parse_arguments() -> argparse.Namespace:
         type=int,
         help='Initial delay between reconnections, doubles after each unsuccessful attempt and is reset after success.',
     )
+    parser.add_argument(
+        'concent_public_key',
+        type=str,
```

Comment:
It would be nice to have some short version of arguments, like `-c` for that `--concent-cluster-port`, and similar for others parameters.

Reply:
:+1: done